app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,26 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/parking_walpaper_main"
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/title_main_activity"
+        style="@style/TitleMainActivityStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/title_main"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/parkingSizeBtn"
+        style="@style/ConfigureParkingSizeBtnMainActivityStyle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="@string/txt_main_activity_parking_size_btn_label"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
```

Comment:
Maybe you can use match_parent width to avoid the constraints

Reply:
Done - I also changed the hardcodes and the button layout like we talked